grillz_calculator/main.py

```diff
@@ -58,6 +63,9 @@ def checking_inputs(inputs):
             except:
                 return 'wrong_input'
 
+    if any(inputs[idx] for idx in [0, 1, 4, 9]) == '':
```

Comment:
'Magic values' anti pattern. It will be better to move it in separate constants with meaning name.